app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> Any:
```

Comment:
You could consider adding a decorator from `functools` here.

Reply:
Done. Thanks for comment.